browse/scripts/topicsParser.py

```diff
@@ -13,6 +13,7 @@ def gen_topics(self, file):
                     if (temp_topic == ''):
                         pass
                     else:
+                        temp_topic = temp_topic.replace(' ', '-')
```

Comment:
This, I understand, is done to handle topics who have a `space` in their names. Is is possible to keep the spaces intact and yet make the script work? The spaces as such have no impact on the manually created links. They work fine on the website.

Reply:
also the `human-body.md` file which was generated after running the script turned out to be an empty file, but it has a link in the topics-list file. 

On the other hand, the `human body.md` file (with a `space`) was generated correctly but it did have an associated link in the topics-list file.

Reply:
Sure I can leave spaces in if it works fine on the website, it just appears in the file as if it is not working. By the way, is there a way for contributors to access the website? I think it'd be helpful I think if I could see how it all works, thanks!

Reply:
The website is accessible via its link given on the repo page. That's how even I access it, there is no special access. 

If you wish to check the output locally on your machine before it is deployed on the website, there are 2 ways I think. One is to open the MD files in a markdown editor/viewer. Other is to deploy the website locally on your machine. I'll paste the link to show how it's done in sometime.

Reply:
Ok good to know thank you I'm not sure how I didn't notice the website link all this time.

Reply:
The website however is generated from the `main` branch. 
So you won't be able to see your changes there till I merge them onto `main`. 

But you can always test your changes by locally hosting the website.